layers/meta-resin-beaglebone/recipes-core/images/resin-image.inc

```diff
@@ -1,4 +1,5 @@
 IMAGE_FSTYPES_append_beaglebone = " resinos-img resinos-img.bmap"
+IMAGE_FSTYPES_append_beaglebone-pocket = " resinos-img resinos-img.bmap"
```

Comment:
You can remove the `resinos-img.bmap` image type addition since this functionality is not present yet (nor being worked on afaicr)